templates/common/page_start.html

```diff
@@ -18,6 +18,9 @@
   $else:
     <title>Weasyl</title>
 
+  <!-- If we have JavaScript enabled, remove the no-js class -->
+  <script>document.documentElement.classList.remove("no-js")</script>
```

Comment:
If this also adds the `js` class, line 184 of `scripts.js` can be removed:

```js
$('html').removeClass('no-js').addClass('js');
```

Reply:
I don't believe ``yall.js`` adds a ``js`` class, when looking at [the source](https://github.com/malchata/yall.js/blob/master/dist/yall.js), or the [usage information](https://github.com/malchata/yall.js).

This specific line was added per the directive on ['What about users without JavaScript?'](https://github.com/malchata/yall.js#what-about-users-without-javascript).

Reply:
Sorry, I mean “if you change it to also add a `js` class”. Right now it removes the `no-js` class, which is half of what line 184 does.